src/var/flat.js

```diff
@@ -0,0 +1,14 @@
+define( [
+	"./arr",
+	"./concat"
+], function( arr, concat ) {
+	"use strict";
```

Comment:
Please de-indent the contents of this function. We plan to migrate to ES modules and we'd like to avoid massive indentation changes then.